app/pdfs/task_pdf_loader.rb

```diff
@@ -0,0 +1,23 @@
+class TaskPdfLoader < Prawn::Document
```

Comment:
class is printing tasks of Project. Why it has name "Task..."
rename to something like ProjectTasksPdfReport